core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -476,7 +476,8 @@ object KafkaConfig {
   val ControllerMessageQueueSizeDoc = "The buffer size for controller-to-broker-channels"
   val DefaultReplicationFactorDoc = "default replication factors for automatically created topics"
   val ReplicaLagTimeMaxMsDoc = "If a follower hasn't sent any fetch requests or hasn't consumed up to the leaders log end offset for at least this time," +
-  " the leader will remove the follower from isr"
+  " the leader will remove the follower from isr. replica.lag.time.max.ms should be smaller than request.timeout.ms" +
+  " to reduce message duplication caused by unnecessary producer retry."
```

Comment:
I thought it is good to have to in both places. But it is not necessary. I will remove it.